storage/sql/accounts/get.sql

```diff
@@ -28,7 +28,20 @@ SELECT
 	"rank",
 	"fees",
 	"rewards",
-	"vote"
+	"vote",
+	"u_username",
+	"u_isDelegate"::int::boolean,
+	"u_secondSignature"::int::boolean,
+	"u_nameexist"::int::boolean as "u_nameExist",
+	"u_multimin" as "u_multiMin",
+	"u_multilifetime" as "u_multiLifetime",
+	"u_balance",
+	case
+	when
+		"producedBlocks" + "missedBlocks" = 0 then 0
```

Comment:
```suggestion
		"isDelegate" = 0 then 0
```

Reply:
If the a delegate account has `producedBlocks` and `missedBlocks` equal to zero, the `else` statement will fail due to division by zero.

Reply:
I tend to believe on our application logic that produced blocks and missed blocks will only be incremented for delegate accounts.

Reply:
But you can still have a delegate account with `producedBlocks` and `missedBlocks` equal to zero